resources/views/emails/mship/account/welcome.blade.php

```diff
@@ -69,10 +65,10 @@
 
 <p>
     The UK TeamSpeak Server, Slack and Forum - these are used for voice and written communications for the entire UK community. Feel free to join the community and ask questions and meet others:
-    <li> TeamSpeak &amp; Slack - {!! link_to("https://www.vatsim.uk/tsreg/", "https://www.vatsim.uk/tsreg/") !!}</li>
+    <li> TeamSpeak &amp; Slack - {!! link_to_route('site.community.teamspeak') !!}</li>
```

Comment:
Can we change this to be just TeamSpeak please? I have it split in my Discord PR to have a link to Slack rather than claiming it as the same page as TeamSpeak. 

For Slack, we ping them a separate invite anyway.

Reply:
Done!